SickSangHae/Model/Categories.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Categories.swift
+//  SickSangHae
+//
+//  Created by user on 2023/07/23.
+//
+
+import Foundation
+
+enum Categories: Int16 {
+    case Unknown = 0
+    case Meat = 1
+    case Bread = 2
+    case Grains = 3
```

Comment:
Swift에서 case는 lower camel case를 선호합니다... 🐫 ...
```suggestion
    case unknown = 0
    case meat = 1
    case bread = 2
    case grains = 3
```

Reply:
안녕하세요 영!
우선은 코드리뷰를 해주셔서 감사합니다. 이런 코드리뷰 덕분에 조금씩 더 swift스러운 코드를 쓸 수 있는것 같아서 기분이 너무 좋아요 :)

말씀하신 enum 의 convention 에 대한 부분을 찾아봤습니다.
https://docs.swift.org/swift-book/documentation/the-swift-programming-language/enumerations/
이 공식 documentation 에 보면 Lower Camel Case 를 쓰라고 명시되어있지는 않지만, case 들은 전부 다 lower camel case 로 쓰여있어서 암묵적으로는 Lower Camel Case 가 맞는것 같습니다.

그러나 https://forums.swift.org/t/swift-case-conventions-for-enums/703 이 문서는 좀 오래됐긴 했지만 개발자들이 Upper CamelCase 가 좋냐, Lower Camel Case 가 좋냐로 논의가 나오는데 Upper Camel Case 의 장점도 있어보였습니다.
Jonathan Hise Kaldma 라는 분의 의견에 따르면
1. 다른 언어에서는 Upper Camel Case 를 사용하기때문에 사용자들이 Swift 에서만 이질감을 느끼지 않게 하기 위해서 Upper Camel Case 가 좋다.
2. Enum 은 그 자체로 돋보여야하기 때문에 enum 의 이름을 UpperCase 로 사용하는 언어들이 많다. case 를 Lower Camel Case 는 돋보이게 해야한다는 것에 위배된다.
3. Upper Camel Case 로 사용하는것이 바로 enum 임을 알 수 있기 때문에 가독성이 좋다. 예를 들어 suit = .Hearts 는 바로 enum 임을 알 수 있지만, suit = .hearts 는 property 인지, property 라면 어디서 오는지 등이 헷갈릴 수 있다.
라는 의견이 있었습니다.

개인 혹은 팀의 코딩 스타일에 따라서 달라질 것 같긴 한데, 저는 영이 말씀해주시고 공식문서에 있는것처럼 Lower Camel Case 를 앞으로 사용하도록 하겠습니다 :)